skyportal/models/obj.py

```diff
@@ -266,6 +266,10 @@ class Obj(Base, conesearch_alchemy.Point):
         default=False,
         doc="Boolean indicating whether the object is a moving object.",
     )
+    mpc_name = sa.Column(
+        sa.String,
+        doc="Minor planet center name(s).",
```

Comment:
Is there a situation where multiple names exist? If so, are they just going to be saved as a comma separated list in a string?

Reply:
Changing this to be singular